### PR TITLE
Fixed Issue #34 - Caused incorrect RA/DEC scale

### PR DESCRIPTION
**Pull request by trq5014**
Changed conversion of arcs to dec in ImageCube.to_FITS() from arcs * 180/pi to arcs/3600
Added test to ensure correct RA/DEC scale in the created fits images

### CODE REVIEW

File: test/images_test.py [hunk 1]
```diff
@@ -48,6 +50,26 @@ def test_imagecube_grad(coords):
     # https://github.com/pytorch/pytorch/issues/50014
     loss.backward()
 
+# test for proper fits scale
+def test_imagecube_tofits(coords):
```
Comment (iancze) [inline test/images_test.py:54]: You might try using the "tmp_path" parameter provided as part of Pytest, and then saving the FITS file to this temporary directory (e.g., https://github.com/MPoL-dev/MPoL/blob/main/test/images_test.py#L108)

The tmp_path is a [Pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html) that creates a temporary directory for the duration of the run. That way, when the tests are complete, the directory is automatically cleaned up.

File: test/images_test.py [hunk 2]
```diff
@@ -48,6 +50,26 @@ def test_imagecube_grad(coords):
...
+    # sending the basecube through the imagecube
+    imagecube.forward(bcube.forward())
+
+    # creating output fits file with name 'test_cube_fits_file39.fits'
+    # file will be deleted after testing
+    imagecube.to_FITS(fname='test_cube_fits_file39.fits', overwrite=True)
+
+    # inputting the header from the previously created fits file
+    fits_header = fits.open('test_cube_fits_file39.fits')[0].header
+    os.remove('test_cube_fits_file39.fits')
```
Comment (iancze) [inline test/images_test.py:70]: For example, if you implement the tmp_path, then this remove line wouldn't be necessary.

**Review by iancze — CHANGES_REQUESTED** [review]

File: test/images_test.py (hunk 2 above)
Comment (trq5014) [reply test/images_test.py]: Just updated this and pushed it to my directory. I think everything looks good now!

**Comment by iancze** [comment]
Nice work, thanks!

**Pull request merged**